drizzlepac/runastrodriz.py

```diff
@@ -453,29 +452,36 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
                                              force_alignment=force_alignment,
                                              find_crs=True, **adriz_pars)
 
-        if align_with_apriori:
-            _trlmsg = _timestamp('Starting alignment with a priori solutions')
-            _trlmsg += __trlmarker__
-            if align_dicts is not None:
-                find_crs = not align_dicts[0]['alignment_verified']
-            else:
-                find_crs = False
+        if align_dicts is not None:
+            find_crs = not align_dicts[0]['alignment_verified']
+        else:
+            find_crs = False
+
+        # run updatewcs with use_db=True to insure all products have
+        # have a priori solutions as extensions
+        # This is NOT done in the apriori sub-directory in order to insure that all
+        # relevant solutions from the astrometry database get appended to the input FLT/FLC files.
+        #
```

Comment:
Since I was momentarily confused regarding the comment beginning at Line 473, perhaps it would be good to add a comment before the use of updatewcs reminding folks that all of the "returned" WCS solutions are based upon the "newest" distortion model by default (all_wcs=False).